packages/web3-providers-ws/package.json

```diff
@@ -16,7 +16,7 @@
     "eventemitter3": "^4.0.0",
     "underscore": "1.9.1",
     "web3-core-helpers": "1.2.8",
-    "websocket": "^1.0.31"
+    "ws": "^7.3.0"
```

Comment:
According to the documentation, this module doesn't work in the browser.

I think we need two implementations of this provider, one for node using `ws`, and another one for the browser using the native `WebSocket` implementation. These can share most of their code.

The browser version should be declared in the package.json's `browser` field: https://docs.npmjs.com/files/package.json#browser